06-regression.Rmd

```diff
@@ -918,6 +919,8 @@ Here is another example: a not-so-great medical doctor goes through their medica
 
 ```{r moderndive-figure-causal-graph-2, echo=FALSE, fig.align='center', fig.cap="Does sleeping with shoes on cause headaches?"}
 knitr::include_graphics("images/flowcharts/flowchart.010-cropped.png")
+knitr::include_graphics("images/ss/207112555.png")
```

Comment:
Same as with wrapper function picture. Edit `images/flowcharts/flowchart.key` and output to `.png` in `images/flowcharts/flowchart.010-cropped.png` so that output looks like 
<img width="853" alt="Screen Shot 2019-06-17 at 5 47 13 PM" src="https://user-images.githubusercontent.com/4574203/59639116-3b3bb980-9128-11e9-98bd-d5abc43d600e.png">